### PR TITLE
Set 2nd tof shooter can id and updated for it in code

### PR DESCRIPTION
**Pull request by Naher123**
Set 2nd tof shooter can id and updated for it in code

### CODE REVIEW

File: ted/src/main/java/frc/robot/subsystems/FeederSubsystem.java [hunk 1]
```diff
@@ -69,8 +69,8 @@ public FeederMode getFeederMode() {
    * @return true if the note is detected
    */
   public boolean isDunkerNoteDetected(){
```
Comment (josieammer) [inline ted/src/main/java/frc/robot/subsystems/FeederSubsystem.java]: we want to remove this method.

File: ted/src/main/java/frc/robot/subsystems/FeederSubsystem.java [hunk 2]
```diff
@@ -80,8 +80,8 @@ public boolean isDunkerNoteDetected(){
    * @return true if the note is detected
    */
   public boolean isShooterNoteDetected(){
-    if (shooterBeambreakSensor != null){
-      return shooterBeambreakSensor.isNoteDetected();
+    if (firstShooterBeambreakSensor != null){
```
Comment (josieammer) [inline ted/src/main/java/frc/robot/subsystems/FeederSubsystem.java]: for this method, we want to return true if either tof is true. 
if both are installed return firstShooterBeamBreakSensor.isNoteDetected() OR secondShooterBeamBreakSensor.isNoteDetected() 
If first is installed, return firstShooterBeamBreakSensor.isNoteDetected()
if second is installed, return
secondShooterBeamBreakSensor.isNoteDetected()
else return false.

**Review by josieammer — CHANGES_REQUESTED** [review]
need some changes to the functionality.

File: ted/src/main/java/frc/robot/subsystems/FeederSubsystem.java (hunk 1 above)
Comment (Naher123) [reply ted/src/main/java/frc/robot/subsystems/FeederSubsystem.java]: removed

File: ted/src/main/java/frc/robot/subsystems/FeederSubsystem.java (hunk 2 above)
Comment (Naher123) [reply ted/src/main/java/frc/robot/subsystems/FeederSubsystem.java]: updated method

**Review by josieammer — APPROVED** [review]
Observed Naher testing on robot. It works as intended. When both or only one of the sensors is powered, the note is detected and intake stops. 
Code changes look good.

**Pull request merged**